assets/templates/kubernetes.container/configs.yml

```diff
@@ -0,0 +1,49 @@
+- name: cloud.availabilit_zone
+  value: "europe-west1-d"  
+- name: agent.id
+  value: "12f376ef-5186-4e8b-a175-70f1140a8f30"
+- name: agent.name
+  value: "kubernetes-scale-123456"
+- name: agent.ephemeral_id  
+  value: "f94220b0-2ca6-4809-8656-eb478a66c541"
+- name: agent.version
+  value: "8.7.0"
+- name: agent.snasphost
+  value: false
+- name: metricset.period
+  value: 10000
+- name: event.duration
+  range:
+    min: 1
+    max: 4000000
+- name: faults
+  range:
+    min: 0
+    max: 500000
+- name: kubernetes.container.rootfs.inodes.used
+  cardinality:
```

Comment:
is it correct to have `cardinality` here instead of `range`?

btw, `cardinality` is going to be refactored at https://github.com/elastic/elastic-integration-corpus-generator-tool/pull/92

you'll have to change your configuration to `cardinality: 100000`

Reply:
Agree changed to range